src/bzflag/ServerLink.h

```diff
@@ -63,6 +63,7 @@ class ServerLink
     const PlayerId& getId() const;
     const char*     getVersion() const;
 
+    void        send(const void* msg, int len, bool udp);
```

Comment:
I would not have this method either public, or an overload. The other methods in this class are based around a message id (`code`) whereas this sends a raw buffer. Given the constraints on the API, allowing an arbitrary blob to be sent seems like a bad idea.

I might name this something like `sendBuffer`.

You might also encapsulate the branches of this method into `sendUdpBuffer` and `sendTcpBuffer`, but that's subjective